deeplake/core/dataset/dataset.py

```diff
@@ -2093,7 +2095,7 @@ def pytorch(
         *args,
         **kwargs,
     ):
-        """Converts the dataset into a pytorch Dataloader.
+        """Creates a dataset PyTorch Dataloader from the Deep Lake dataset. During iteration, the data from all tensors will be streamed on-the-fly from the storage location.
```

Comment:
maybe `Creates a PyTorch Dataloader from the Deep Lake dataset`?

Reply:
@activesoull Fixed. Thx for the suggestion